app/src/main/java/com/nervousfish/nervousfish/activities/BluetoothConnectionActivity.java

```diff
@@ -29,17 +28,17 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-import java.io.IOException;
 import java.util.HashSet;
-import java.util.List;
 import java.util.Set;
 
 /**
  * An {@link Activity} that establishes and manages a bluetooth connection.
  * It shows a screen with the Bluetooth devices with which the user is paired and other Bluetooth devices
  * that are detected near the device of the user.
  */
-@SuppressWarnings("PMD.ExcessiveImports") // Uses many Android and utility classes
+@SuppressWarnings({"PMD.ExcessiveImports", "PMD.TooManyMethods"})
+// 1. Uses many Android and utility classes
+// 2. The amount of methods is not too much at this moment
```

Comment:
I think this depends on the largeness of the methods. This class has a lot of small methods where it is no problem, but somewhere else it could be better to have this PMD error.